modules/ngap_module/NgapNames.h

```diff
@@ -58,6 +58,10 @@ constexpr static auto const USE_DMRPP_CACHE = "NGAP.UseDMRppCache";
 constexpr static auto const DMRPP_CACHE_THRESHOLD = "NGAP.DMRppCacheSize";
 constexpr static auto const DMRPP_CACHE_SPACE = "NGAP.DMRppCachePurge";
 
+constexpr static auto const DMRPP_FILE_CACHE_THRESHOLD = "NGAP.DMRppFileCacheSize"; // in MB
```

Comment:
At this point the server is littered with BESKeys for size with unit that are not contained in the key name but rather in a comment. And we have bytes, kb. mb, etc. 

I think we should start adding the units to the key name to remove the confusion:
```
NGAP.DMRppFileCacheSize.MB
NGAP.DMRppFileCachePurge.MB
```

Reply:
Yes. I'll do that.